### PR TITLE
Fixed issues

### PR DESCRIPTION
**Pull request by 0xVikas**
Issue #80 , Timings changed for VK(Girls)

Issue #81 

Issue #82 , Frontend form is sending the bhavan name along with other fields

Issue #89 , Users can now clear the image selected

### CODE REVIEW

File: complaints/static/main.js [hunk 1]
```diff
@@ -13,20 +13,34 @@ function urg_fun() {
...
+		document.getElementById('hiddentime2').value = '13:30:00';
+		}
 		
 		if(checkBox2.checked == true){
 		document.getElementById('hiddentime').value = '04:30:00';
-		document.getElementById('hiddentime2').value = '05:30:00';
-		document.getElementById('slot_1').setAttribute('disabled','');}
-		else{document.getElementById('slot_1').removeAttribute('disabled');}
+		document.getElementById('hiddentime2').value = '05:00:00';
+		}
```
Comment (Aviral14) [inline complaints/static/main.js]: The timings should be 16:30 and 17:00

File: complaints/templates/complaints/handle_complaints.html [hunk 2]
```diff
@@ -39,8 +39,7 @@
                <tbody bgcolor="white">
                   <tr>
                      <td>{{ comp.category }}</td>
-
-                     <td>{{ comp.room_no }}</td>
+                     <td>{{ comp.room_no }}, {{ comp.bhavan }}</td>
```
Comment (Aviral14) [inline complaints/templates/complaints/handle_complaints.html]: Bhavan should come before Room Number.

Comment (Aviral14) [reply complaints/templates/complaints/handle_complaints.html]: Moreover, make sure that the complaint ids are also being displayed to users in the my complaint tab(I guess this is not being done) and to all the staff in All Complaints tab.

File: complaints/static/main.js [hunk 3]
```diff
@@ -13,20 +13,34 @@ function urg_fun() {
...
-		else{document.getElementById('slot_2').removeAttribute('disabled');}
+		document.getElementById('hiddentime2').value = '13:30:00';
+		}
 		
 		if(checkBox2.checked == true){
 		document.getElementById('hiddentime').value = '04:30:00';
-		document.getElementById('hiddentime2').value = '05:30:00';
-		document.getElementById('slot_1').setAttribute('disabled','');}
-		else{document.getElementById('slot_1').removeAttribute('disabled');}
+		document.getElementById('hiddentime2').value = '05:00:00';
```
Comment (Kushagra-0801) [inline complaints/static/main.js]: These are supposed to be pm
`16:30:00` and `17:00:00`

**Pull request merged**